filebeat/tests/system/test_reload_inputs.py

```diff
@@ -278,11 +278,6 @@ def test_reload_same_config(self):
         with open(self.working_dir + "/configs/input.yml", 'w') as f:
             f.write(inputConfigTemplate.format(self.working_dir + "/logs/test.log"))
 
-        # Make sure error shows up in log file
```

Comment:
Why this test had to change? It was specifically designed to check that a new file cannot be harvested before the old one is finished.

Reply:
@ruflin this PR makes sure that such a scenario doesnt occur as part of the config reloading.

Reply:
I think it still should and can occur especially for example if the same file shows up in 2 different configs.

Reply:
I guess that is true but from the perspective of this test case by making the call blocking, this can never happen.